keyboards/1up60rgb/keymaps/raffle/keymap.c

```diff
@@ -0,0 +1,130 @@
+//****************************************************************************//
+// raffle's keymap for the 1up60rgb.                                          //
+//   emulates my pok3r layout and adds RGB control + firmware reset/debug     //
+//   layers                                                                   //
+//****************************************************************************//
+
+//************************ dependencies + definitions ************************//
+#include "1up60rgb.h"
```

Comment:
Could you replace this with `#include QMK_KEYBOARD_H`?

Reply:
Sure thing -- fixed now.